DOCPLAN-100: Vale updates for /virt/support/virt-troubleshooting.adoc (#108585)
openshift/openshift-docs: modules/virt-common-error-messages.adoc

= Common error messages

[role="_abstract"]

> To diagnose issues and troubleshoot problems, you can review common error messages in your {VirtProductName} logs. 

Reviewer: Suggestion to reduce fluff:

```suggestion
Troubleshoot {VirtProductName} by reviewing common error messages found in the logs. 
```

Author: Done.
